root/artist_credit/ArtistCreditLayout.js

```diff
@@ -54,19 +56,22 @@ const ArtistCreditLayout = ({
   >
     <div id="content">
       <h1>
-        <a href={'/artist-credit/' + artistCredit.id}>
-          {texp.l(
+        <ArtistCreditUsageLink
+          artistCredit={artistCredit}
+          content={texp.l(
             'Artist credit “{artist_credit}”',
             {artist_credit: reduceArtistCredit(artistCredit)},
           )}
-        </a>
+        />
       </h1>
       <Tabs>
         {tabLinks.map(link => (
           <li className={page === link[0] ? 'sel' : ''} key={link[0]}>
-            <a href={'/artist-credit/' + artistCredit.id + link[0]}>
-              {link[1]()}
-            </a>
+            <ArtistCreditUsageLink
+              artistCredit={artistCredit}
+              content={link[1]()}
+              subPath={link[0].replace(/^\/?(.*)/, '$1')}
```

Comment:
This was a bit confusing at first. You can write this so that it's just replacing the leading slash with an empty string (`?` isn't needed either, since the replacement is a no-op in that case).
```suggestion
              subPath={link[0].replace(/^\//, '')}
```

Reply:
Though it might also be better to move this replacement into the `ArtistCreditUsageLink` component.

Reply:
In most cases I'd expect there's no need to do a replacement, so having it in the component seems strange to me. Made the change, at least.